### PR TITLE
Allow users to edit its own profile 

### PR DESCRIPTION
**Pull request by MathiasLantean**

### CODE REVIEW

File: profile/tests/test_views.py [hunk 1]
```diff
@@ -177,6 +190,23 @@ def test_reset_password_does_not_match(self):
         response = self.client.post(url, data, format='json')
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_update_user_profile_ok(self):
+        user_to_edit = CommonUserFactory(gender=User.Gender.GENDER_FEMALE)
+        self.client.force_login(user_to_edit)
+        url = reverse('rest_user_details')
+        new_first_name = 'new_first_name'
+        new_last_name = 'new_last_name'
```
Comment (kaozdl) [inline profile/tests/test_views.py]: you can parametrize these

File: profile/tests/test_views.py [hunk 2]
```diff
@@ -33,6 +33,8 @@ def test_sign_up_user_ok(self):
         url = reverse("auth:rest_register")
         data = {
             'email': self.user_email,
+            'first_name': 'NameTest',
+            'last_name': 'LastNameTest',
```
Comment (kaozdl) [inline profile/tests/test_views.py]: and these

**Review by kaozdl — COMMENTED** [review]
looks good

File: profile/tests/test_managers.py [hunk 3]
```diff
@@ -10,17 +10,26 @@ class UserManagerTestCase(TestCase):
     def setUpTestData(cls):
         cls.user_email = 'test@test.com'
         cls.user_password = 'test123455'
+        cls.user_first_name = 'userTestName'
+        cls.user_last_name = 'userTestLastName'
```
Comment (guilleijo) [inline profile/tests/test_managers.py]: Use snake case

File: profile/tests/test_views.py (hunk 2 above)
Comment (guilleijo) [inline profile/tests/test_views.py]: Same here

**Pull request merged**